src/main/java/uk/ac/cam/cl/dtg/segue/etl/ETLConfigurationModule.java

```diff
@@ -112,11 +111,8 @@ protected void configure() {
     @Singleton
     private static ContentMapper getContentMapper() {
         if (null == mapper) {
-            Reflections r = new Reflections("uk.ac.cam.cl.dtg.segue");
+            Reflections r = new Reflections("uk.ac.cam.cl.dtg");
             mapper = new ContentMapper(r);
-
-            mapper.registerJsonTypes(new SegueConfigurationModule().getContentDataTransferObjectMap());
-
```

Comment:
Same as the main Guice config; now scan `segue` and `isaac` together and don't have the weird divide for the ContentMapper.